### PR TITLE
Added error messages in frontend

### PR DESCRIPTION
**Pull request by ThomasRoehl**
please check if something else has to be done

### CODE REVIEW

**Review by vickyokrm — COMMENTED** [review]
'Next' button on the login screen is missing.

File: src/settings/login.js [hunk 1]
```diff
@@ -86,7 +95,16 @@
 
             $('#navToLogin').on('click', function () {
                 var login = '#login';
-                showView(login);
+                if (validateUrl($('#server').val()).status != 200) {
+                    const error = {};
```
Comment (vickyokrm) [inline src/settings/login.js]: Also ES5 here.

**Review by vickyokrm — CHANGES_REQUESTED** [review]

**Comment by ThomasRoehl** [comment]
The login button is hidden as long as no valid url is entered ... see regex validation.

@janrudolph I see 4 options here, what do you prefer:
- button is always visible and clickable
- button is greyed out and only clickable if regex validation does match
- button is hidden as long as regex validation does not match
- we remove the button completely and proceed as soon as the regex validation matches

**Comment by janrudolph** [comment]
> * button is greyed out and only clickable if regex validation does match

this one

**Comment by ThomasRoehl** [comment]
should look like this now:
image

**Comment by vickyokrm** [comment]
`https://team.assistify-test....../asdadaada/adaadadada`  is validated as correct url.

**Comment by vickyokrm** [comment]
Authentication validation is shown as `error` instead of actual error message.

**Comment by vickyokrm** [comment]
`https://asdaa/asd`  is validated as a right url.

**Review by vickyokrm — CHANGES_REQUESTED** [review]
Messages snack bar view is not visible in the outlook.

**Comment by ThomasRoehl** [comment]
For me it is visible in the webapp and in the local app.
Might be a problem on ie11 but i can not test that

**Comment by ThomasRoehl** [comment]
> `https://asdaa/asd` is validated as a right url.

This should be fixed now. please try again

**Pull request merged**